slate-social-media/src/Components/Cards/Profilepagepostcard.css

```diff
@@ -0,0 +1,187 @@
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e9ebee;
+}
+
+.f-card {
+  background-color: white;
+  /*height:400px;*/
+  width: 502px;
```

Comment:
Suggestion - For bigger pixel values than 10px use rem, that will help in responsive design too later on